app/_src/guides/otel-metrics.md

```diff
@@ -0,0 +1,155 @@
+
+Most important in this configuration is `pipelines` section:
+
+```yaml
+pipelines:
+  metrics:
+    receivers: [otlp]
+    processors: [memory_limiter, batch]
+    exporters: [prometheus]
+```
+
+In this basic guide we will focus only on collecting metrics, but this can be also easily configured to collect traces and logs.
+We use `otlp` receiver to accept metrics pushed from dataplane proxies. 
+
+Then we have basic recommended processors to limit memory usage and to 
+process metrics in batch. You can filter, modify and do more with [available processors](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor).
+
+Last part is `exporters` section. You can export metrics to multiple destination like Prometheus, Datadog, Grafana Cloud and more. Full list of available exporters can be found [here]().
+We will use Prometheus exporter for now.
+
+With configuration in place we can install OpenTelemetry collector:
+
+```shell
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+helm install --namespace mesh-observability opentelemetry-collector open-telemetry/opentelemetry-collector -f values.yaml
+```
+
+## Configure Prometheus to scrape metrics from OpenTelemetry collector
+
+We need to update `prometheus-server` ConfigMap and add `scrape_configs` entry:
+```yaml
+- job_name: "opentelemetry-collector"
+  scrape_interval: 15s
+  static_configs:
+    - targets: ["opentelemetry-collector.mesh-observability.svc:8889"]
+```
+
+Prometheus will automatically pick up this config and start scraping OpenTelemetry collector.
+
+## Enable OpenTelemetry metrics and check results
+
+By now we have installed and configured all needed observability tools: OpenTelemetry collector, Prometheus and Grafana.
+We can now apply [MeshMetric policy](/docs/{{ page.version }}/policies/meshmetric):
+
+```shell
+echo "
+apiVersion: kuma.io/v1alpha1
+kind: MeshMetric
+metadata:
+  name: 'otel-metrics'
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  default:
+    backends:
+      - type: OpenTelemetry
+        openTelemetry:
+          endpoint: opentelemetry-collector.mesh-observability.svc:4317
+" | kubectl apply -f -
+```
+
+This policy will configure all dataplane proxies in `default` Mesh to collect and push metrics to OpenTelemetry collector. 
+
+To check results we need to log into Grafana. First enable port forward to Grafana GUI:
+
+```shell
+kubectl port-forward svc/grafana -n mesh-observability 3000:80
+```
+
+Then navigate to browser `http://localhost:3000` and check `Dataplane` dashboard. You should see something similar to:
+
+<center>
+<img src="/assets/images/guides/otel-metrics/grafana-dataplane-view.png" alt="Dataplane Grafana dashboard"/>
+</center>
+
+## Next steps
+
+* Further explore [MeshMetric policy](/docs/{{ page.version }}/policies/meshmetric)
```

Comment:
should we link guides how to export this to hosted solutions?

Reply:
good idea, I've added links to tutorials